### ISSUE TITLE
Unify User properties for various User Cards

### ISSUE THREAD

**Issue by me-andre** [alkem-io/client-web#2819]
As a developer I want to be able to easily replace one UserCard component by another without re-mapping properties to a new format, especially given that the data is mostly the same.
The solution would be that all UserCards, such as: ContributorCard, UserCard and LeadUserCard - all take User properties in the same format.
I propose that for that format we use prop names as they are defined in the GraphQL schema; for the long paths, we just skip/join fragments (such as user.profile.avatar.uri becomes user.avatarUri).

Please note that this change would not be trivial since a lot of utils rely on those types (such as mapping functions doing remapping without explicitly specifying the target type), so an automatic refactoring is only possible to some extent.

**Comment by techsmyth** [comment]
@me-andre yes good to standardize this. The Author model is a bit in this direction - and in the branch you just reviewed you can see buildAuthorFromUser that converts from api to model. Not strongly typed at the moment so that needs to be added.
